bundles/org.openhab.binding.remoteopenhab/README.md

```diff
@@ -24,39 +26,73 @@ You will find in the inbox one discovery thing per remote server interface.
 So if your remote server has one IPv4 address and one IPv6 address, you will discover two things in the inbox.
 Just choose one of the two things.
 
+Once a bridge thing representing a remote openHAB server is created, all things from this remote server are discovered when you scan for new things.
+
 ## Binding Configuration
 
 The binding has no configuration options, all configuration is done at Thing level.
 
 ## Thing Configuration
 
-The thing has the following configuration parameters:
+The `server` thing has the following configuration parameters:
+
+| Parameter          | Required | Description                                                                                                       |
+|--------------------|----------|-------------------------------------------------------------------------------------------------------------------|
+| host               | yes      | The host name or IP address of the remote openHAB server.                                                         |
+| useHttps           | no       | Set it to true in case you want to use HTTPS to communicate with the remote openHAB server. Default is false.     |
+| port               | yes      | The HTTP port to use to communicate with the remote openHAB server. Default is 8080.                              |
+| trustedCertificate | no       | Set it to true in case you want to use HTTPS even without a valid SSL certificate provided by your remote server. |
+| restPath           | yes      | The subpath of the REST API on the remote openHAB server. Default is /rest                                        |
+| token              | no       | The token to use when the remote openHAB server is setup to require authorization to run its REST API.            |
+
+The `thing` thing has the following configuration parameters:
+
+| Parameter     | Required | Description                                 |
+|---------------|----------|---------------------------------------------|
+| thingUID      | yes      | The thing UID in the remote openHAB server. |
+| buildChannels | no       | If set to true, a trigger channel will be automatically created and linked to each trigger channel from the remote thing. Default is true. |
```

Comment:
I feel that this param should be renamed to `buildTriggerChannels` since from the description this config only affects trigger channels and not all channels.

Reply:
Renamed